yb-voyager/cmd/exportData.go

```diff
@@ -605,6 +638,15 @@ func exportDataOffline(ctx context.Context, cancel context.CancelFunc, finalTabl
 	}
 
 	source.DB().ExportDataPostProcessing(exportDir, tablesProgressMetadata)
+	if source.DBType == POSTGRESQL {
```

Comment:
Ideally,  we should do this in ExportDataPostProcessing of PostgresDB itself. Is there anything blocking that? 

THe datafiledescriptor is written by pg_dump layer itself i believe. 
Not so sure about the exportSnapshotStatusFile

Reply:
Yes, you're right dataFileDescriptor for all non-dbzm cases is written in exportDataPostProcessing. But when I was trying to make changes there, It was a bit more code duplication that way as I already have the rename function in this code path, So I realised this way is easier and less complicated to just update these things once written.